firestore-shorten-urls-dynamic-links/extension.yaml

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: firestore-shorten-urls-dynamic-links
+displayName: Shorten URLs with Dynamic Links
+specVersion: v1beta
+version: 0.1.0
+
+description: >
+  Shortens URLs written to a specified Cloud Firestore collection using
+  Firebase Dynamic Links.
+
+license: Apache-2.0
+billingRequired: false
```

Comment:
can you make billingRequired true and update functions to Node 10? Might as well do it now and not have to do it later.